srcpkgs/ueberzug/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'ueberzug'
+pkgname=ueberzug
+version=18.1.3
+revision=1
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+makedepends="python3-devel libX11-devel libXext-devel"
+depends="libXext python3-xlib python3-pillow-simd python3-docopt python3-psutil python3-attrs"
```

Comment:
You don't need to [explicitly depend on libXext](https://github.com/void-linux/void-packages/blob/master/Manual.md#explain_depends).